doc/en/how-to/parametrize.rst

```diff
@@ -29,10 +29,6 @@ pytest enables test parametrization at several levels:
 
 .. regendoc: wipe
 
-
-
-    Several improvements.
-
```

Comment:
Another leftover I noticed when reviewing the original commit. I think it got hidden as a comment because it's intended under the `.. regendoc: wipe`.